Simperium/build.gradle

```diff
@@ -97,69 +94,16 @@ public final class Version {
 
 tasks.preBuild.dependsOn versionConfig
 
-afterEvaluate {
-
-    configurations {
-        archives {
-            extendsFrom configurations.default
-        }
-    }
-
-    signing {
-        sign configurations.archives
-    }
-
-    uploadArchives {
-        configuration = configurations.archives
-        repositories {
-            mavenDeployer {
-                beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
-
-                if (project.hasProperty("ossrhUsername") && project.hasProperty("ossrhPassword")) {
-                    repository(url: "https://oss.sonatype.org/service/local/staging/deploy/maven2/") {
-                      authentication(userName: ossrhUsername, password: ossrhPassword)
-                    }
-
-                    snapshotRepository(url: "https://oss.sonatype.org/content/repositories/snapshots/") {
-                      authentication(userName: ossrhUsername, password: ossrhPassword)
-                    }
-                }
-
-
-                pom.project {
-                    name "Simperium Android Library"
-
-                    groupId 'com.simperium.android'
-                    artifactId 'simperium'
-
-                    packaging 'aar'
-                    description 'Android Library for building applications using Simperium'
-
-                    url 'https://github.com/simperium/simperium-android'
-                    scm {
-                        connection 'scm:git:https://github.com/simperium/simperium-android.git'
-                        developerConnection 'scm:git:https://github.com/simperium/simperium-android.git'
-                        url 'https://github.com/simperium/simperium-android.git'
-                    }
-
-                    licenses {
-                        license {
-                            name 'The MIT License (MIT)'
-                            url 'http://opensource.org/licenses/MIT'
-                        }
-                    }
-
-                    developers {
-
-                        developer {
-                            id 'beaucollins'
-                            name 'Robert Collins'
-                            email 'robert@automattic.com'
-                        }
-
-                    }
-                }
-            }
-        }
-    }
+publish {
+    artifactId = 'simperium'
+    userOrg = 'simperium'
+    groupId = 'com.simperium.android'
+    uploadName = 'simperium'
+    description = 'Android Library for building applications using Simperium'
+    publishVersion = android.defaultConfig.versionName
```

Comment:
Looking at previous version names here https://repo1.maven.org/maven2/com/simperium/android/simperium/ - we should probably change how we set `publishVersion` here, maybe having a simple string "0.7.2" for now?

Reply:
Using a string like `'0.7.2'` works for me.  Would `publishVersion = project.version` work too?  Either way is fine with me.

Reply:
`project.version` and `android.defaultConfig.versionName` is the same string, and comes from the `git describe` command which return the latest tag + number of changes + a hash. We could get the latest tag only and even check the current commit was tagged and fail if not.

I prefer having a simple string here as it's more flexible and we won't release it often. Updated in 46d2e92